packages/core/test/tracing/index_test.js

```diff
@@ -67,21 +78,27 @@ mochaSuiteFn('[UNIT] tracing/index', function () {
   });
 
   it('deactivate instrumentation via config', () => {
-    initAndActivate({ tracing: { disabledTracers: ['grpc', 'kafkajs'] } });
+    initAndActivate({ tracing: { disabledTracers: ['grpc', 'kafkajs', 'aws-sdk/v2/index'] } });
     expect(activateStubGrpc).to.not.have.been.called;
     expect(activateStubGrpcJs).to.have.been.called;
     expect(activateStubKafkaJs).to.not.have.been.called;
     expect(activateStubRdKafka).to.have.been.called;
     expect(initStubGrpc).not.to.have.been.called;
+
+    expect(initAwsSdkv2).not.to.have.been.called;
+    expect(activateAwsSdkv2).not.to.have.been.called;
   });
 
   it('deactivate instrumentation via env', () => {
     process.env.INSTANA_DISABLED_TRACERS = 'grpc';
     initAndActivate({});
+
     expect(activateStubGrpc).to.not.have.been.called;
     expect(activateStubGrpcJs).to.have.been.called;
     expect(activateStubKafkaJs).to.have.been.called;
     expect(activateStubRdKafka).to.have.been.called;
+    expect(activateAwsSdkv2).to.have.been.called;
```

Comment:
_thought:_ We could use the same pairs of init and activate checks as in the test case 'deactivate instrumentation via config' here. On the other hand, the test setup only deactivates `grpc`, so to be honest, I do not remember why we check for all the other instrumentations here as well 🤔   I think just having expectations for `grpc` and maybe `grpcjs` for their respective init and activate methods would be enough?